dockstore-webservice/src/main/java/io/dockstore/webservice/helpers/CheckUrlHelper.java

```diff
@@ -26,36 +26,30 @@
 import java.net.http.HttpRequest;
 import java.net.http.HttpResponse;
 import java.util.ArrayList;
-import java.util.HashSet;
-import java.util.Iterator;
 import java.util.List;
-import java.util.Map;
 import java.util.Optional;
 import java.util.Set;
 import java.util.stream.Collectors;
 import javax.ws.rs.core.UriBuilder;
-import org.json.JSONArray;
-import org.json.JSONObject;
 import org.slf4j.Logger;
 import org.slf4j.LoggerFactory;
-import org.yaml.snakeyaml.Yaml;
 
-public final class CheckUrlHelper {
+public final class CheckUrlHelper implements CheckUrlInterface {
```

Comment:
Since we use the `Helper` suffix to denote helper classes, and this class is no longer a helper, but rather a specialization of `CheckUrlInterface`, suggest we use the typical naming scheme, i.e. `LambdaCheckUrl` or similar.